build.gradle

```diff
@@ -2,7 +2,7 @@
 
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
That’s better!
Since I pushed again, please confirm. :smiley:

https://github.com/ogapants/androidtv-Leanback/commit/43beb22d886a5c92747e36d0d00d46e659d76786